packages/BiostructureViewer/src/utils/ngl-gl-doc-service.ts

```diff
@@ -0,0 +1,210 @@
+import * as grok from 'datagrok-api/grok';
+import * as ui from 'datagrok-api/ui';
+import * as DG from 'datagrok-api/dg';
+
+import {_package} from '../package';
+import * as NGL from 'NGL';
+import {NglGlServiceBase, NglGlTask} from '@datagrok-libraries/bio/src/viewers/ngl-gl-viewer';
+
+
+export class NglGlDocService implements NglGlServiceBase {
+  private readonly nglDiv: HTMLDivElement;
+  private readonly ngl: any;
```

Comment:
Is it really "any"? Can we get types for NGL?

Reply:
I added some typization with NGL.Stage class at typings/index.d.ts module.